docker-compose.yaml

```
@@ -679,6 +679,8 @@ services:
       - BAG_API_CLIENT_MP_REST_URL=${BAG_API_CLIENT_MP_REST_URL:-https://api.bag.kadaster.nl/lvbag/individuelebevragingen/v2/}
       - BAG_API_KEY=${BAG_API_KEY:-fakeBagApiKey}
       - BRP_API_CLIENT_MP_REST_URL=http://brp-personen-mock:5010/haalcentraal/api/brp
+      - BRP_DOELBINDING_ZOEKMET=BRPACT-ZoekenAlgemeen
```

Guess this is ok too but for most other variables we get them from the environment variables (i.e. our `.env.tpl` file normally depending on how you start up the thing). You can still have the same value as default. Maybe a little overkill this way but at least this makes sure we use the same value when starting up ZAC from IntelliJ vs from our Docker Compose scripts.

```suggestion
      - BRP_DOELBINDING_ZOEKMET=${BRP_DOELBINDING_ZOEKMET:-BRPACT-ZoekenAlgemeen}
```